website/thaliawebsite/views.py

```diff
@@ -51,3 +54,19 @@ def get_context_data(self, **kwargs) -> dict:
         )
 
         return context
+
+
+class RateLimitedPasswordResetView(PasswordResetView):
+    @method_decorator(ratelimit(key="ip", rate="5/h"))
```

Comment:
Wouldn't
```python
    @method_decorator(ratelimit(key="ip", rate="5/h"), name="post")
    class RateLimitedPasswordResetView(PasswordResetView):
        pass
```
be just a bit cleaner?
But also I don't really care if you'd rather not do the effort of changing it

Reply:
I thought it didn't work like this but now I see it does